Source/Core/Core/ActionReplay.cpp

```diff
@@ -598,13 +600,13 @@ static bool ZeroCode_MemoryCopy(const u32 val_last, const ARAddr& addr, const u3
   LogInfo("Src Address: %08x", addr_src);
   LogInfo("Size: %08x", num_bytes);
 
-  if ((data & ~0x7FFF) == 0x0000)
+  if ((data & 0xFF0000) == 0)
```

Comment:
This, too, was due to a misinterpretation of the spec. Furthermore, this check made the "Memory Copy With Pointers Support" branch completely unreachable.